static/components/ExpectedChanges.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import api from 'app/api';
+
+import TaskSummary from './TaskSummary';
+
+function ExpectedChanges({app, env}) {
```

Comment:
Naming it this since I'm also going to have this do another API call to get us the list of commits